src/Libraries/Nop.Services/Common/Caching/AddressCacheEventConsumer.cs

```diff
@@ -11,13 +11,17 @@ namespace Nop.Services.Common.Caching
     public partial class AddressCacheEventConsumer : CacheEventConsumer<Address>
     {
         /// <summary>
-        /// Clear cache data
+        /// Clear cache by entity event type
         /// </summary>
         /// <param name="entity">Entity</param>
+        /// <param name="entityEventType">Entity event type</param>
         /// <returns>A task that represents the asynchronous operation</returns>
-        protected override async Task ClearCacheAsync(Address entity)
+        protected override async Task ClearCacheAsync(Address entity, EntityEventType entityEventType)
```

Comment:
I don't quite understand why it is not necessary to update the cache when adding a new address?

Reply:
When we add a new address, it is not yet associated with any customer

Reply:
Perhaps, but I'm not completely sure that this is the case, especially considering that the standard behavior can be changed by some kind of plugin. I don't think the idea of removing the insert event is safe

Reply:
It seems to me that we will gain more performance if we do not remove all cached addresses from all users each time an address is added.

Reply:
I would agree with this thesis if it were not a potential problem, in such a case as this, it is better to delete the data than not to delete it at the right time

Reply:
Ok, let's leave it as it was, probably this issue needs to be approached more comprehensively.